packages/postgres-database/src/simcore_postgres_database/models/projects.py

```diff
@@ -141,6 +141,18 @@ class ProjectType(enum.Enum):
         default=False,
         doc="If true, the project is by default not listed in the API",
     ),
+    sa.Column(
```

Comment:
A project belongs to a workspace and if a workspace is deleted, all associated projects are deleted. Is that the case?

Reply:
Yes